From 96c03e3dcca301a851d52e01b6ad9705bff11569 Mon Sep 17 00:00:00 2001
From: Emil Renner Berthing <emil.renner.berthing@canonical.com>
Date: Tue, 14 Jul 2026 15:45:14 +0800
Subject: [PATCH 089/334] FROMLIST: riscv: dts: thead: Add IO labels for the IO
 expansion on I2C1

Add labels for GPIOs on the IO expansion chip on the I2C1 bus.

Signed-off-by: Emil Renner Berthing <emil.renner.berthing@canonical.com>
[Icenowy: Splitted off the patch adding all 3 IO expansions]
Signed-off-by: Icenowy Zheng <zhengxingda@iscas.ac.cn>

Link: https://lore.kernel.org/all/20260714074515.1959352-1-zhengxingda@iscas.ac.cn/
Signed-off-by: Mingcong Bai <jeffbai@aosc.io>
---
 arch/riscv/boot/dts/thead/th1520-lichee-pi-4a.dts | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/arch/riscv/boot/dts/thead/th1520-lichee-pi-4a.dts b/arch/riscv/boot/dts/thead/th1520-lichee-pi-4a.dts
index 74b120c6b063a..78f5979a56c14 100644
--- a/arch/riscv/boot/dts/thead/th1520-lichee-pi-4a.dts
+++ b/arch/riscv/boot/dts/thead/th1520-lichee-pi-4a.dts
@@ -128,6 +128,14 @@ ioexp2: gpio@18 {
 		reg = <0x18>;
 		gpio-controller;
 		#gpio-cells = <2>;
+		gpio-line-names = "",
+				  "cam0_reset",
+				  "cam1_reset",
+				  "cam2_reset",
+				  "wl_host_wake",
+				  "bt_resetn",
+				  "",
+				  "bt_host_wake";
 	};
 };
 
-- 
2.55.0

